### PR TITLE
Add more logic to the question page

### PR DESCRIPTION
**Pull request by odrusso**
Absolutely no style or adherence to the designs yet
Just trying to get the bits into place

### CODE REVIEW

File: web/src/components/Question.tsx [hunk 1]
```diff
@@ -1,5 +1,60 @@
...
+// TODO Refactor this into a models area
+type QuestionTypeDTO = {
+    type: string,
+    id: string
+}
 
 export const Question = (): JSX.Element => {
-    return (<h1>Question page</h1>)
+
+    const listOfQuestions: QuestionTypeDTO[] = [
```
Comment (odrusso) [inline web/src/components/Question.tsx:11]:
```suggestion
    // TODO: Pull this dynamically from API
    const listOfQuestions: QuestionTypeDTO[] = [
```

File: web/mockServiceWorker.js [hunk 2]
```diff
@@ -0,0 +1,338 @@
+/* eslint-disable */
```
Comment (odrusso) [inline web/mockServiceWorker.js:1]: This file is autogenerated

File: web/package.json [hunk 3]
```diff
@@ -100,5 +100,8 @@
     "testMatch": [
       "**/src/**/?(*.)+(spec|test).[jt]s?(x)"
     ]
+  },
+  "msw": {
+    "workerDirectory": "."
   }
-}
+}
```
Comment (odrusso) [inline web/package.json:107]:
```suggestion
}

```

File: web/mockServiceWorker.js [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+/* tslint:disable */
+
+/**
+ * Mock Service Worker (0.34.0).
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
+ */
+
+const INTEGRITY_CHECKSUM = 'f0a916b13c8acc2b526a03a6d26df85f'
```
Comment (ashtonmoomoo) [inline web/mockServiceWorker.js:11]: This feels like something that should be treated carefully? Maybe in an environment variable or something

File: web/src/mocks/handlers.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        )
+    }),
+
+    rest.get('/api/v1/questions/:questionType/:questionID', (req, res, ctx) => {
+        return res(
+            ctx.status(200),
+            ctx.json({
+                "name": `${req.params['questionType']} ${req.params['questionID']}`,
+                "description": "Solve a linear equation.",
+                "latex": "a * x + b = 0"
```
Comment (ashtonmoomoo) [inline web/src/mocks/handlers.tsx:18]: I've currently got this as `"question": "a x + b = 0"` in `questions.py`

Comment (odrusso) [reply web/src/mocks/handlers.tsx:18]: Sick okay

File: web/mockServiceWorker.js (hunk 4 above)
Comment (odrusso) [reply web/mockServiceWorker.js:11]: Maybe, it's just for local dev and was autogenerated so I'm too fussed about it

File: web/src/mocks/handlers.tsx (hunk 5 above)
Comment (odrusso) [reply web/src/mocks/handlers.tsx:18]: Will fix in the next PR

**Pull request merged**